test/epgsql_SUITE.erl

```diff
@@ -847,6 +848,17 @@ array_type(Config) ->
         Select(jsonb, [<<"{}">>, <<"[]">>, <<"1">>, <<"1.0">>, <<"true">>, <<"\"string\"">>, <<"{\"key\": []}">>])
     end).
 
+record_type(Config) ->
+    Module = ?config(module, Config),
+    epgsql_ct:with_connection(Config, fun(C) ->
+        {ok, _, [{{1, 2}}]} = Module:equery(C, "select (1,2)", []),
+        {ok, _, [{{1, {2, 3}}}]} = Module:equery(C, "select (1, (select (2,3)))", []),
+        {ok, _, [{{1, [2, 3]}}]} = Module:equery(C, "select (1, '{2,3}'::int[])", []),
+        {ok, _, [
+            {{<<"test">>,[{1,<<"one">>},{2,<<"two">>}]}}
+        ]} = Module:equery(C, "select ('test', ARRAY(select (id, value) from test_table1))", [])
```

Comment:
nit: would be cool to have some comments describing what are we asserting in each case. Like "record inside record", "array nested in record" etc.

Reply:
Fixed. Review please